Formula/p/pigz.rb

```diff
@@ -39,7 +39,7 @@ def install
     test_data = "a" * 1000
     (testpath/"example").write test_data
     system bin/"pigz", testpath/"example"
-    assert (testpath/"example.gz").file?
+    assert_predicate testpath/"example.gz", :file?
```

Comment:
Maybe `assert_path_exists` is better? If I remember correctly..

Reply:
`assert_path_exists` is not equivalent, since that's a replacement for
```
assert_predicate testpath/example.gz, :exist?
```